recipes/google-cloud-cpp-compute/meta.yaml

```diff
@@ -0,0 +1,119 @@
+{% set version = "2.17.0" %}
+
+package:
+  name: google-cloud-cpp-compute
+  version: {{ version }}
+
+source:
+  url: https://github.com/googleapis/google-cloud-cpp/archive/v{{ version }}.tar.gz
+  sha256: 8cb87ec2947e691a7f8631877e78453bcfda51b3d3cd4940794a376741888d37
+  patches:
+    # This is an upstream fix to allow sharding google/cloud/oauth2
+    - patches/0001-Backport-12911-from-upstream.patch
+
+build:
+  number: 0
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - ninja
+    - libgrpc
+    - libprotobuf
+  host:
+    - libgoogle-cloud-devel =={{ version }}
+    - libabseil
+    - libcurl
+    - libgrpc
+    - libprotobuf
+    - openssl
+    - nlohmann_json
+    - zlib # Needed by gRPC
+
+outputs:
+  - name: libgoogle-cloud-compute
+    script: install-libgoogle-cloud.sh   # [unix]
+    script: install-libgoogle-cloud.bat  # [win]
+    build:
+      run_exports:
+        - libgoogle-cloud =={{ version }}
```

Comment:
This should probably just run-export itself, otherwise we probably get a circularity problem when trying to build `libgoogle-cloud` that includes `libgoogle-cloud-compute`

Reply:
I apologize, but that went over my head.  The thing that will depend on `libgoogle-cloud-compute` is `libgoogle-cloud-all`?

Reply:
Run-exports are for capturing the relationship that compiling against a given shared library generates a run-time requirement for the same (or an ABI-compatible) library, in order to be able to resolve the symbols.

In other words, compiling against `libgoogle-cloud-compute[-devel]` should generate a run-export on `libgoogle-cloud-compute`. In a way, it's harmless though, because either way, a package compiled against `libgoogle-cloud-compute` will end up with a `libgoogle-cloud` runtime requirement, as `*-compute` itself needs that (and it gets generated as a run-export from the `libgoogle-cloud-devel` host-dep).